scss/_base_grid-definitions.scss

```diff
@@ -5,6 +5,31 @@
   padding-right: 0;
 }
 
+// Apply grid properties for a given breakpoint
+@mixin vf-b-grid-template-impl($breakpoint, $columns) {
```

Comment:
I intend it to mean "implementation" - I am using `vf-b-grid-template` as an abstraction over the implementation because it has a conditionally-applied media query, and the styles applied are many lines long, so an abstraction saves us from some repetition:

I.E., we get this:

```scss
// Apply grid properties for a given breakpoint
@mixin vf-b-grid-template-impl($breakpoint, $columns) {
  grid-gap: 0 map-get($grid-gutter-widths, $breakpoint);
  grid-template-columns: repeat($columns, minmax(0, 1fr));
  // Expose the number of columns available for child elements as a CSS var, so that columns can span full grid width without increasing selector specificity\
  --vf-grid-columns: #{$columns};

  & > * {
    grid-column-end: span$columns;
  }
}

// Apply grid properties for a given breakpoint. If a min_width is provided, wrap the properties in a media query
@mixin vf-b-grid-template($breakpoint, $columns, $min_width: null) {
  @supports (display: grid) {
    @if $min_width {
      @media (min-width: $min_width) {
        @include vf-b-grid-template-impl($breakpoint, $columns);
      }
    } @else {
      @include vf-b-grid-template-impl($breakpoint, $columns);
    }
  }
}
```
Instead of this:
```scss
// Apply grid properties for a given breakpoint. If a min_width is provided, wrap the properties in a media query
@mixin vf-b-grid-template($breakpoint, $columns, $min_width: null) {
  @supports (display: grid) {
    @if $min_width {
      @media (min-width: $min_width) {
         grid-gap: 0 map-get($grid-gutter-widths, $breakpoint);
         grid-template-columns: repeat($columns, minmax(0, 1fr));
          --vf-grid-columns: #{$columns};
        
          & > * {
            grid-column-end: span $columns;
          }
      }
    } @else {
         grid-gap: 0 map-get($grid-gutter-widths, $breakpoint);
         grid-template-columns: repeat($columns, minmax(0, 1fr));
          --vf-grid-columns: #{$columns};
        
          & > * {
            grid-column-end: span $columns;
          }
    }
  }
}
```